staticx/hooks/glibc.py

```diff
@@ -13,6 +14,8 @@ def process_glibc_prog(sx):
     try:
         nssfix = copy_asset_to_tempfile(LIBNSSFIX, debug=sx.debug,
                 prefix='libnssfix-', suffix='.so')
+        umask=os.umask(0o755)
+        os.chmod(nssfix.name, 0o755&~umask)
```

Comment:
FYI: There is already a `make_executable` function in `staticx.utils`.

Reply:
FYI: Your new code shouldn't be inside of the existing `try`/`except KeyError` block.